src/sdi-refresh-monitor.c

```diff
@@ -645,65 +675,37 @@ void sdi_refresh_monitor_init(SdiRefreshMonitor *self) {
   configure_snapd_monitor(self);
 }
 
-static void ignore_snap_cb(GObject *obj, const gchar *snap_name,
-                           SdiRefreshMonitor *self) {
+/**
+ * This CB must be called whenever the user presses the "Don't remind me
```

Comment:
CB => callback? Seems unnecessary to abbreviate.

Reply:
Absolutely right. Fixing it...